bokeh/document.py

```diff
@@ -752,10 +752,6 @@ def validate(self):
             None
 
         '''
-        # logging.basicConfig is a no-op if there's already
-        # some logging configured. We want to make sure warnings
-        # go somewhere so configure here if nobody has.
-        logging.basicConfig(level=logging.INFO)
```

Comment:
Right you've added them to `serve.py` so they will be configured (presumably) when people execute `bokeh serve app.py`. These warnings need to show up *always*. E.g. in the jupyter notebook, in random scripts people create. None of that kind if library usage goes through `bokeh serve` (or any `bokeh` subcommand)